### PR TITLE
Thriftsync plugin to sync handlers with thrift file

### PR DESCRIPTION
**Pull request by anuptalwalkar**
GFM-162
Executing following go generate command creates ``server/handlers.go`` with necessary struct and methods.
``//go:generate thriftrw "--plugin=thriftsync" <thrift_filename>``

thriftsync provides sync between handler code and thrift file for any method signature change. The plugin looks for updating any new methods that are added.

### CODE REVIEW

File: examples/keyvalue/server/handlers.go [hunk 1]
```diff
@@ -23,9 +23,10 @@ package main
 import (
 	"sync"
 
+	kvs "go.uber.org/fx/examples/keyvalue/kv/keyvalueserver"
+
```
Comment (sectioneight) [inline examples/keyvalue/server/handlers.go]: nit: put this next to the other keyvalue import, not in a separate group

File: modules/rpc/thriftrw-plugin-thriftsync/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+**New handler generation**
+
+```thrift
+service TestService {
+    string testFunction(1: string param)
+}
+```
+
+```lang=go
```
Comment (sectioneight) [inline modules/rpc/thriftrw-plugin-thriftsync/README.md]: This is phab remarkup. Omit `lang=`

File: modules/rpc/thriftrw-plugin-thriftsync/README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+```thrift
+service TestService {
+    string testFunction(1: string param)
+
+    string newtestFunction(1: string param)
+}
+```
+
+```lang=go
```
Comment (sectioneight) [inline modules/rpc/thriftrw-plugin-thriftsync/README.md]: remove lang=

File: modules/rpc/thriftrw-plugin-thriftsync/README.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+```thrift
+service TestService {
+    string testFunction(1: string param)
+
+    string newtestFunction(1: string param, 2: string parameter2)
+}
+```
+
+```lang=go
```
Comment (sectioneight) [inline modules/rpc/thriftrw-plugin-thriftsync/README.md]: same

**Comment by anuptalwalkar** [comment]
> Looks good, just nits. This stuff was previously reviewed elsewhere, right? This is just pulling it into our repo?

Yes, Only update here is to use thriftrw/plugin APIs instead of build as a CLI.

**Pull request merged**